### PR TITLE
Generate the Create gRPC method

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: entproto/cmd/protoc-gen-entgrpc/cast.go [hunk 1]
```diff
@@ -51,3 +52,27 @@ func (g *serviceGenerator) castToProtoFunc(fld *entproto.FieldMappingDescriptor)
...
 	}
 }
+
+func (g *serviceGenerator) castToEntFunc(fd *entproto.FieldMappingDescriptor) (interface{}, error) {
+	var fld *gen.Field
+	if fd.IsEdgeField {
+		fld = fd.EntEdge.Type.ID
+	} else {
+		fld = fd.EntField
+	}
```
Comment (a8m) [inline entproto/cmd/protoc-gen-entgrpc/cast.go:62]: Not mandatory, but I usaully do:

```suggestion
	fld := fd.EntField
	if fd.IsEdgeField {
		fld = fd.EntEdge.Type.ID
	}
```

File: entproto/cmd/protoc-gen-entgrpc/methods.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import (
+	"entgo.io/ent/entc/gen"
+	"google.golang.org/protobuf/compiler/protogen"
+)
+
+var (
+	camel = gen.Funcs["camel"].(func(string) string)
+)
+
+func (g *serviceGenerator) generateCreateMethod() error {
```
Comment (a8m) [inline entproto/cmd/protoc-gen-entgrpc/methods.go:26]: Yay!!! 😍

File: entproto/internal/entprototest/fieldmapper_test.go [hunk 3]
```diff
@@ -41,6 +41,8 @@ func (suite *AdapterTestSuite) TestFieldMap() {
 	assert.EqualValues("BlogPosts", blogPosts.PbStructField())
 	assert.False(blogPosts.IsIDField)
 	assert.True(blogPosts.IsEdgeField)
+	assert.EqualValues("Id", blogPosts.EdgeIDPbStructField())
```
Comment (a8m) [inline entproto/internal/entprototest/fieldmapper_test.go:44]: Is it common to write the identifier field this way, and not `ID`?

**Review by a8m — APPROVED** [review]
Wow @rotemtam! That's really great.


Thanks a lot for pushing this

File: entproto/internal/entprototest/fieldmapper_test.go (hunk 3 above)
Comment (rotemtam) [reply entproto/internal/entprototest/fieldmapper_test.go:44]: Yes this ist he way protoc generated Go code from a field named `id`

**Pull request merged**